src/data/shopping-list.ts

```diff
@@ -25,7 +27,9 @@ export const completeItem = (
   itemId: number,
```

Comment:
I realize now that this is a bit weird, we shouldn't have 2 methods that call same function. The functions in `data/*` should be modeled after the APIs, not how it's used in our web components.

We should merge these two methods, like this: https://github.com/home-assistant/home-assistant-polymer/blob/8ad5280501e3e5b91a48f740df39956bb3c11616/src/panels/config/cloud/data.ts#L7-L18

(Yes, this method should be under `data/cloud.ts` which I'll fix in a future PR)

Reply:
I’ll do so in my next pr. Thanks!